### PR TITLE
[Doc] Translate the visualization.md

### PR DESCRIPTION
**Pull request by matrixgame2018**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMDet3D.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/user_guides/visualization.md [hunk 1]
```diff
@@ -1 +1,174 @@
...
+然后你可以找到他们本地的路径和使用下面的脚本文件对其进行可视化：
+
+```python
+import mmcv
+import os.path as osp
+import torch
+
+# `PixelData` 是 MMEngine 中用于定义的像素级注释或预测的数据结构的模块。
+# 请参考下面的MMEngine数据结构教程文件：
+# https://github.com/open-mmlab/mmengine/tree/main/docs/en/advanced_tutorials/data_element.md
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:105]:
```suggestion
# https://mmengine.readthedocs.io/zh_CN/latest/advanced_tutorials/data_element.html#pixeldata
```

File: docs/zh_cn/user_guides/visualization.md [hunk 2]
```diff
@@ -1 +1,174 @@
...
+
+from mmengine.structures import PixelData
+
+# `SegDataSample`是不同组件之间的数据结构接口
+# 在 MMSegmentation 中定义，它包括ground truth、预测和语义分割的预测逻辑。
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:110]:
```suggestion
# `SegDataSample` 是在 MMSegmentation 中定义的不同组件之间的数据结构接口，
# 它包括 ground truth、语义分割的预测结果和预测逻辑。
```

File: docs/zh_cn/user_guides/visualization.md [hunk 3]
```diff
@@ -1 +1,174 @@
...
+gt_sem_seg = PixelData(**gt_sem_seg_data)
+data_sample = SegDataSample()
+data_sample.gt_sem_seg = gt_sem_seg
+
+seg_local_visualizer = SegLocalVisualizer(
+    vis_backends=[dict(type='LocalVisBackend')],
+    save_dir=save_dir)
+
+# 数据集的元信息通常包括类名的`classes`和
+# `palette` 用于可视化每个前景的颜色。
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:143]:
```suggestion
# 数据集的元信息通常包括类名的 `classes` 和
# 用于可视化每个前景颜色的 `palette` 。
```

File: docs/zh_cn/user_guides/visualization.md [hunk 4]
```diff
@@ -1 +1,174 @@
...
+# 所有类名和调色板都可以在文件中定义：
+# https://github.com/open-mmlab/mmsegmentation/blob/1.x/mmseg/utils/class_names.py
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:145]:
```suggestion
# 所有类名和调色板都在此文件中定义：
# https://github.com/open-mmlab/mmsegmentation/blob/1.x/mmseg/utils/class_names.py
```

File: docs/zh_cn/user_guides/visualization.md [hunk 5]
```diff
@@ -1 +1,174 @@
...
+    palette=[[128, 64, 128], [244, 35, 232], [70, 70, 70],
+             [102, 102, 156], [190, 153, 153], [153, 153, 153],
+             [250, 170, 30], [220, 220, 0], [107, 142, 35],
+             [152, 251, 152], [70, 130, 180], [220, 20, 60],
+             [255, 0, 0], [0, 0, 142], [0, 0, 70],
+             [0, 60, 100], [0, 80, 100], [0, 0, 230],
+             [119, 11, 32]])
+
+# 当`show=True`时，直接显示结果，
+# 当 `show=False`时，结果将保存在本地目录文件夹中。
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:162]:
```suggestion
# 当 `show=False`时，结果将保存在本地文件夹中。
```

File: docs/zh_cn/user_guides/visualization.md [hunk 6]
```diff
@@ -1 +1,174 @@
...
+
+seg_local_visualizer.add_datasample(out_file, image,
+                                    data_sample, show=False)
+```
+
+可视化后的图像结果和它的对应的 ground truth 图像放在了 `./work_dirs/vis_data/vis_image/` 的路径当中，它的名字是：`out_file_cityscapes_0.png` ：
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:168]:
```suggestion
可视化后的图像结果和它的对应的 ground truth 图像可以在 `./work_dirs/vis_data/vis_image/` 路径找到，文件名字是：`out_file_cityscapes_0.png` ：
```

**Review by xiexinch — COMMENTED** [review]
Hi @matrixgame2018,
Thanks for your contribution, please fix the lint error :)

**Comment by matrixgame2018** [comment]
How to fix this error？

**Comment by MeowZheng** [comment]
> 
> 
> How to fix this error？

please try to run `pre-commit run --all-files`

File: docs/zh_cn/user_guides/visualization.md [hunk 7]
```diff
@@ -1 +1,173 @@
...
+    palette=[[128, 64, 128], [244, 35, 232], [70, 70, 70],
+             [102, 102, 156], [190, 153, 153], [153, 153, 153],
+             [250, 170, 30], [220, 220, 0], [107, 142, 35],
+             [152, 251, 152], [70, 130, 180], [220, 20, 60],
+             [255, 0, 0], [0, 0, 142], [0, 0, 70],
+             [0, 60, 100], [0, 80, 100], [0, 0, 230],
+             [119, 11, 32]])
+
+# 当`show=True`时，直接显示结果，
+# # 当 `show=False`时，结果将保存在本地文件夹中。
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:161]:
```suggestion
# 当 `show=False`时，结果将保存在本地文件夹中。
```

File: docs/zh_cn/user_guides/visualization.md [hunk 8]
```diff
@@ -1 +1,173 @@
...
+
+```shell
+tensorboard --logdir work_dirs/test_visual/20220810_115248/vis_data
+```
+
+## 数据和结果的可视化
+
+### 模型测试或验证期间的可视化数据样本
+
+MMSegmentation 提供了 `SegVisualizationHook` ，它是一个 可以用于可视化 ground truth 和在模型测试和验证期间的预测分割结果的[钩子](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/hook.html) 。 它的配置在 `default_hooks` 中，更多详细信息请参见 [执行器教程](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/runner.html)。
```
Comment (xiexinch) [inline docs/zh_cn/user_guides/visualization.md:51]:
```suggestion
MMSegmentation 提供了 `SegVisualizationHook` ，它是一个可以用于可视化 ground truth 和在模型测试和验证期间的预测分割结果的[钩子](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/hook.html) 。 它的配置在 `default_hooks` 中，更多详细信息请参见 [执行器教程](https://mmengine.readthedocs.io/zh_CN/latest/tutorials/runner.html)。
```

**Review by xiexinch — APPROVED** [review]

**Review by MeowZheng — APPROVED** [review]

**Pull request merged**